### ISSUE TITLE
Accessing Ink/Stitch: non-blocking Ink/Stitch panel

### ISSUE THREAD

**Issue by agentogden** [inkstitch/inkstitch#2606]
Would be nice if the inkstitch menu was easier to get to.  an always open tool palette would be nice.

**Comment by kaalleen** [comment]
I'm afraid we are bound to the limitations of the inkscape extension system. There is hope though, that it could be possible in some future.

If you don't want to click through the whole menu all the time, you could easier set shortcut keys to access Ink/Stitch extensions faster.
An other (or additional options) would be, to setup your own ui file for the Inkscape menu and place Ink/Stitch extensions at top level. It's a bit of manual works though and you'll have to remember to update your menu with every update (Inkscape and Ink/Stitch) by yourself. If you want to go down that road I can provide you more detailed information.

**Comment by LochanLoch** [comment]
Yeah a short HowTo :-) would be nice. I am a bit out of way don't working much with that cold.

**Comment by kaalleen** [comment]
Ok... In you Inkscape `Preferences > System` have a look for `Inskcape data` folder. From there you may have to enter the inkscape folder and then the `ui` folder. Inside the ui folder find the file `menus.ui` copy that to the `user ui` folder (also found in preferences system). Adapt the copied menus.ui file in the user ui folder to your liking.

**Comment by kaalleen** [comment]
I think we have that now with the extension gallery.
